tern/classes/package.py

```diff
@@ -43,7 +43,7 @@ def license(self):
         return self.__license
 
     @license.setter
-    def license(self, license):
+    def license(self, license):  # pylint: disable=redefined-builtin
```

Comment:
Again, why did pylint pick up this particular pattern while the other patterns above (like `def version(self, version)` didn't bring up anything?

Reply:
"license" is the built-in here: ```license()``` print the Python license.